style-blogpage.css

```diff
@@ -0,0 +1,662 @@
+}
+
+
+.blog-article {
+    width: 920px;
+    height: 2528px;
+    min-width: 288px;
+    max-width: 1024px;
+    gap: 32px;
+}
+
+.dragon-image {
+    width: 920px;
+    height: 557px;
+}
+
+.category-gaming {
+    width: 122px;
+    height: 48px;
+    padding: 10px 24px;
+    gap: 8px;
+    color: #F5F8FB;
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 28px;
+    background-color: #0265DC;
+    border: 1px solid #0265DC;
+}
+
+.photo-caption, photo-caption1, photo-caption2 {
+    background-color: #0265DC;
+}
+  
+.photo-container {
+    display: flex; 
+    justify-content: center;
+    align-items: center;
+}
+
+.metadata {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 8px;
+    font-size: 14px;
+  }
+  
+  .metadata__artist-name, .metadata__time {
+    width: 84px;
+    height: 20px;
+    gap: 8px;
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 20px;  
+    padding-left: 20px;
+    white-space: nowrap;
+  }
+  
+  .metadata__artist-name {
+    background-image: url(img/user-icon.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
+  .metadata__time {
+    background-image: url(img/calendar-icon.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
+  .basilisks-collection-link {
+    text-decoration: none;
+    color: #78BBFA;
+  }
+
+  .game-developed-link {
+    text-decoration: none;
+    color: #78BBFA;
+  }
+  
+.blog-list-name {
+    list-style: none;
+    width: 135px;
+    height: 136px;
+    gap: 8px;
+}
+
+.blog-list__item {
+    width: 97ox;
+    height: 28px;
+    padding-left: 70px;
+    padding-bottom: 20px;
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    background-image: url(img/half-logo-icon.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+    white-space: nowrap;
+    align-self: center;
+    align-content: center;
+    align-items: center;
+}
+
+.quote {
+    width: 920px;
+    height: 120px;
+    border-radius: 16px;
+    padding: 16px;
+    gap: 8px;
+    margin-top: 100px;
+    background-color: #002754;
+}
+
+.quote_text {
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 30px;
+}
+
+.quote__rating {
+    width: 888px;
+    height: 20px;
+    gap: 8px;
+    padding-left: 40px;
+    background-image: url(img/rating-stars-icon.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+}
+
+
+.nft-binance-link {
+    text-decoration: none;
+    color: #78BBFA;
+}
+
+.binance-link {
+    text-decoration: none;
+    color: #78BBFA;  
+}
+
+.final-links {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 920px;
+    height: 32px;
+    padding-top: 40px;
+}
+.tags,  .share {
```

Comment:
@Alinchik1 you use the same style for both blocks. It's a good chance to use one class name! We can speak about it at the meeting.